### PR TITLE
ui: fix extra newline in dvc version

### PR DESCRIPTION
**Pull request by Suor**
The issue happens when running `dvc version` with no `psutil` installed.

### CODE REVIEW

File: dvc/command/version.py [hunk 1]
```diff
@@ -74,11 +74,11 @@ def run(self):
             root_directory = os.getcwd()
 
         if psutil:
-            info += ("Filesystem type (workspace): {fs_root}").format(
+            info += ("Filesystem type (workspace): {fs_root}\n").format(
                 fs_root=self.get_fs_type(os.path.abspath(root_directory))
             )
 
-        logger.info(info)
+        logger.info(info.rstrip("\n"))
```
Comment (efiop) [inline dvc/command/version.py]: How about we make `info` a list and then just join it properly? 🙂

**Comment by Suor** [comment]
Changed to `"\n".join`, also don't show cache dir fs type unless it exists.

**Comment by efiop** [comment]
@Suor Need to adjust the tests accordingly.

**Comment by Suor** [comment]
-19 lines)

**Comment by efiop** [comment]
Thank you, @Suor !

**Pull request merged**